be/src/tools/meta_tool.cpp

```diff
@@ -122,13 +126,131 @@ void delete_meta(DataDir* data_dir) {
     std::cout << "delete meta successfully" << std::endl;
 }
 
+Status init_data_dir(const std::string dir, DataDir** ret) {
```

Comment:
```suggestion
Status init_data_dir(const std::string& dir, DataDir** ret) {
```

Reply:
and why here use DataDir**?
I think DataDir* or std::unique_ptr<DataDir>* is ok

Reply:
I will change it to unique_ptr